src/Mapbender/CoreBundle/Resources/public/mapbender.element.searchRouter.js

```diff
@@ -398,37 +401,66 @@
         /**
          * Rebuilds result table with search result data.
          *
-         * @param SearchModel       model   Search model
-         * @param FeatureCollection results Search result feature collection
-         * @param object            options Backbone options
+         * @param {SearchModel} model
+         * @param {FeatureCollection} results
+         * @param {Object} options Backbone options (not used?)
          */
         _searchResultsTable: function(model, results, options){
             var headers = this.options.routes[this.selected].results.headers,
                 table = $('.search-results table', this.element),
                 tbody = $('<tbody></tbody>'),
-                layer = this._getLayer(true);
+                layer = this._getLayer(true),
+                self = this;
 
             $('tbody', table).remove();
             layer.removeAllFeatures();
-            features = [];
+            var features = [];
 
             if(results.length > 0) $('.no-results', this.element).hide();
 
-            results.each(function(feature, idx){
+            results.each(function(feature, idx) {
                 var row = $('<tr/>');
                 row.addClass(idx % 2 ? "even" : "odd");
                 row.data('feature', feature);
-                for(var header in headers){
+
+                for (var header in headers) {
                     var d = feature.get('properties')[header];
                     row.append($('<td>' + (d || '') + '</td>'));
                 }
+
                 tbody.append(row);
 
                 features.push(feature.getFeature());
             });
 
             table.append(tbody);
             layer.addFeatures(features);
+
+            $('.search-results tbody tr')
+                .on('click', function () {
+                    var feature = $(this).data('feature').getFeature();
+                    self._highlightFeature(feature, layer, 'select');
+                })
+                .on('mouseenter', function () {
+                    var feature = $(this).data('feature').getFeature();
+
+                    if(feature.renderIntent !== 'select') {
+                        self._highlightFeature(feature, layer, 'temporary');
```

Comment:
Why "temporary"? Why not "highlight"?

Reply:
As I understand "temporary" doesnt need extra care in configuration cause it comes out of OL. See: http://docs.openlayers.org/library/feature_styling.html for reference.

But I would agree with the not optimal wording in view of the given use case.